### PR TITLE
Project 3: Query multiple tables

### PR DESCRIPTION
**Pull request by rskinnerc**
#### In this PR I have:
- [x] Updated the `animals` table to remove the `species` column
- [x] Created the `owners` and species tables
- [x] Added foreign key constraints to the `animals` table referencing the `owners` and `species` tables
- [x] Populated the `owners` and `species` tables
- [x] Assigned owners and species to the existing animals
- [x] Wrote the following queries using `JOIN`:
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,3 +15,40 @@ SELECT AVG(weight_kg) FROM animals;
...
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+SELECT *
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT COUNT(*), species.name
+FROM species
+JOIN animals ON animals.species_id = species.id
```
Comment (Davidosky007) [inline queries.sql:35]: ## * Kindly look into these queries as when i ran them on my end i got a different result compared to your screenshots below ⬇️ are my results screenshots 
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @rskinnerc 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @rskinnerc  👋,

_This is an additional review as a result of the second opinion request._

- The query results have been confirmed from my end and also join a zoom call with the student to confirm the results 👍 

    -  Screenshot (580)


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by mmsesay — APPROVED** [review]
Hi @rskinnerc, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**